### PR TITLE
Documentation: Update japanese translation

### LINKED ISSUE #78301: Add Japanese 2.13 docs
### Summary

Add the translation of the core docs for 2.13 for the Japanese translation.  This entails two steps:

- [x] Test and publish the 2.13 translation files
- [x] Add 2.13 translation link to [the docs.ansible.com](https://docs.ansible.com/core-translated-ja.html)

### Issue Type

Documentation Report

### Component Name

docs/docsite/rst/user_guide/intro_getting_started.rst

### Ansible Version

```console
$ ansible --version
2.13
```

### Configuration

```console
# if using a version older than ansible-core 2.12 you should omit the '-t all'
$ ansible-config dump --only-changed -t all
none
```

### OS / Environment

none

### Additional Information

none

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by Spredzy**
##### SUMMARY

Update Japanese localization for documentation

Fixes #78301

### CODE REVIEW

File: docs/docsite/rst/locales/ja/LC_MESSAGES/getting_started.po [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+msgid ":ref:`vars_and_facts`"
+msgstr ":ref:`vars_and_facts`"
+
+#: ../../rst/getting_started/get_started_playbook.rst:70
+msgid "Learn more about the ``gather_facts`` keyword in playbooks."
+msgstr "Playbook の ``gather_facts`` キーワードの詳細を確認できます。"
+
+#: ../../rst/getting_started/index.rst:5
+msgid "Getting started with Ansible"
+msgstr "MySQL のスタートガイド"
```
Comment (samccann) [inline docs/docsite/rst/locales/ja/LC_MESSAGES/getting_started.po]:
```suggestion
msgstr "Ansible のスタートガイド"
```

File: docs/docsite/rst/locales/ja/LC_MESSAGES/community.po [hunk 2]
```diff
@@ -147,2604 +155,6369 @@ msgid "Policy violations"
...
+msgstr "Ansible で管理されているコレクションへの貢献を歓迎します。これらのコレクションは、ダウンストリームでサポートされている Red Hat 製品の一部であるため、貢献、テスト、およびリリースの基準は、他のコミュニティーコレクションよりも高い場合があります。関連するコミュニティーコレクション (``community.general`` および ``community.network`` など) の要件はそれほど厳しくなく、将来のリリースで Ansible が管理するコレクションの一部になる可能性がある新しい機能に最適な場所です。"
+
+#: ../../rst/community/contributing_maintained_collections.rst:274
+msgid "The following scenarios use the ``arista.eos`` to help explain when to contribute to the Ansible-maintained collection, and when to propose your change or idea to the related community collection:"
+msgstr "以下のシナリオでは、``arista.eos`` を使用して Ansible が管理しているコレクションに貢献するタイミングや、関連するコミュニティーコレクションに変更や考えを提案するタイミングを説明しています。"
+
+#: ../../rst/community/contributing_maintained_collections.rst:277
+msgid "You want to fix a problem in the ``arista.eos`` Ansible-maintained collection. Create the PR directly in the `arista.eos collection GitHub repository <https://github.com/ansible-collections/arista.eos>`_. Apply all the :ref:`merge requirements
```
Comment (samccann) [inline docs/docsite/rst/locales/ja/LC_MESSAGES/community.po:2960]: Not sure why this wasn't translated?

**Review by samccann — COMMENTED** [review]
Just a couple of small comments.

File: docs/docsite/rst/locales/ja/LC_MESSAGES/community.po (hunk 2 above)
Comment (Spredzy) [reply docs/docsite/rst/locales/ja/LC_MESSAGES/community.po:2960]: Not sure what you mean here. This looks translated. Was that the line you were refering to @samccann ?

```
msgid "You want to fix a problem in the ``arista.eos`` Ansible-maintained collection. Create the PR directly in the `arista.eos collection GitHub repository <https://github.com/ansible-collections/arista.eos>`_. Apply all the :ref:`merge requirements <ansible_collection_merge_requirements>`."
msgstr "Ansible が管理する ``arista.eos`` コレクションの問題を修正します。PR を `arista.eos コレクションの GitHub リポジトリー <https://github.com/ansible-collections/arista.eos>`_ に直接作成します。:ref:`マージ要件 <ansible_collection_merge_requirements>` をすべて適用します。"
```

Comment (samccann) [reply docs/docsite/rst/locales/ja/LC_MESSAGES/community.po:2960]: Hmm not sure why it's not picking the correct line. I'm talking about line 2960 
```
#: ../../rst/community/contributor_license_agreement.rst:7
msgid "By contributing you agree that these contributions are your own (or approved by your employer) and you grant a full, complete, irrevocable copyright license to all users and developers of the project, present and future, pursuant to the license of the project."
msgstr "By contributing you agree that these contributions are your own (or approved by your employer) and you grant a full, complete, irrevocable copyright license to all users and developers of the project, present and future, pursuant to the license of the project."
```

**Review by samccann — APPROVED** [review]

**Pull request merged**